server/portal/apps/onboarding/middleware.py

```diff
@@ -32,7 +32,7 @@ def __call__(self, request):
             return HttpResponseRedirect(reverse('portal_accounts:logout'))
 
         # check to see if user setup has finished
-        if not portal_profile.setup_complete:
+        if not portal_profile.setup_complete and reverse('workbench:index') in request.path:
```

Comment:
I can reach this:  `https://cep.dev/accounts/profile/`    which is a problem as its basically allows you access to the workbench. 
so i guess we need to either:
1) check for that route here too
or
2) see if we need to support that `accounts/profile/` for some particular reason.  if we don't, we could turn that route into `workbench/accounts/profile/`

Reply:
Reaching out to @owaisj  on this one to get a sense of that route.

Reply:
Owais and I just discussed briefly.  The `accounts/profile/` path is used to match the url of manage account in cepv1.

Reply:
Perhaps we change to `/workbench/account` - the cepv1 path is arbitrary

Reply:
☝️  that sounds like a good plan.

Reply:
Added ticket -> https://jira.tacc.utexas.edu/browse/FP-734